flow/strict.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  extends: ['./'],
+  rules: {
+    'flowtype/no-flow-fix-me-comments': 'error',
+    'flowtype/no-existential-type': 'error',
+    'flowtype/no-primitive-constructor-types': 'error',
+    'flowtype/no-weak-types': [
+      'error',
+      {
+        any: false,
```

Comment:
This allows the explicit use of `any`, but disallows `Object` and `Function`. The latter two, while more expressive, are generally confusing and make it seem like they're inferring the structure of the named types instead of just aliasing to `any`.